packages/client/src/Call.ts

```diff
@@ -477,7 +477,10 @@ export class Call {
   }) => {
     const response = await this.streamClient.get<GetCallResponse>(
       this.streamClientBasePath,
-      params,
+      {
+        ...params,
+        ring: this.isCreatedByMe ? params?.ring : false,
```

Comment:
I think this "decision" needs to be made at a higher level. I don't think having an API that receives one value but sends another is a good idea.

Can you lift this to the call sites of this method?

Reply:
I agree, will do that, does the function signature work

```
get(apiParams, makeRinging = false)

So in call site,

call.get(undefined, true) would lift the decision to the callsite
```

Reply:
I'd rather have:
```ts
if (makeRinging) {
  call.get({ ring: true });
}

// or

call.get({ ring: makeRinging });
```